librdmacm/CMakeLists.txt

```diff
@@ -42,4 +42,4 @@ install(TARGETS rspreload DESTINATION "${CMAKE_INSTALL_LIBDIR}/rsocket/")
 rdma_install_symlink("librspreload.so" "${CMAKE_INSTALL_LIBDIR}/rsocket/librspreload.so.1")
 rdma_install_symlink("librspreload.so" "${CMAKE_INSTALL_LIBDIR}/rsocket/librspreload.so.1.0.0")
 
-rdma_pkg_config("rdmacm" "" "" "libibverbs")
+rdma_pkg_config("rdmacm" "" "" "")
```

Comment:
Instead of removing these completely they should be a Requires.private: libibverbs ?